src/content/app-platform/getting-started/index.md

```diff
@@ -0,0 +1,260 @@
+---
+linkTitle: Getting started
+title: Getting started with App Platform
+description: Guide to deploying apps using kubectl and the Giant Swarm Management API.
+weight: 20
+menu:
+  main:
+    parent: app-platform
+last_review_date: 2021-04-21
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+user_questions:
+  - How can I deploy an app using an App CR?
+  - How can I configure an App CR?
+  - How can I see which apps are available using the Management API?
+---
+
+# Getting started with App Platform
+
+## Overview
+
```

Comment:
```suggestion
App Platform allows you to manage apps and their configurations (represented by App Custom Resources or App CRs) in multiple clusters, from a single place: the management cluster.
```

Would it be useful to add a one-line reminder of what app platform is?

Reply:
Thanks @cokiengchiara yes the intro needed more context.

I changed it to include Helm. WDYT? 

> _Giant Swarm App Platform_ is built on top of Helm

If this wording is OK I think we should also use it in https://docs.giantswarm.io/app-platform/overview/

Reply:
yes sounds perfect. Great catch on modifying also in that doc. Could you include in this PR? 😅